_config.yml

```diff
@@ -264,6 +264,7 @@ save_scroll: false
 
 # Automatically excerpt description in homepage as preamble text.
 excerpt_description: true
+read_more_btn: true
```

Comment:
oh, it's fine. sorry for the mistake.

Reply:
I mean `excerpt_description` and `auto_excerpt` are related to `more`. so place after the section of `auto_excerpt`  without indents. not meaning of your first commit.

Reply:
```yml
# Automatically excerpt description in homepage as preamble text.
excerpt_description: true

# Automatically Excerpt. Not recommend.
# Please use <!-- more --> in the post to control excerpt accurately.
auto_excerpt:
  enable: false
  length: 150

# read more button, which works only in Excerpt Mode
read_more_btn: true
```
what about this ?

Reply:
yeah, it is what i mean.

Reply:
`which works only in Excerpt Mode` seems no need?

Reply:
OK

Reply:
`If true, the read more button would be displayed in excerpt section`
I prefer like this.

Reply:
thanks for your suggestion